dev-python/asdf/asdf-2.7.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6..7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library for the Advanced Scientific Data Format"
+HOMEPAGE="https://asdf.readthedocs.io/en/latest/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test"
```

Comment:
All flags and options related to testing should be done using `disutil-r1`'s functionality
Reference - https://dev.gentoo.org/~mgorny/python-guide/distutils.html
- No need to manually add a `test` USE flag
- No need to add `RESTRICT` part.
just do something like `distutils_enable_tests pytest`